stdlib/3/builtins.pyi

```diff
@@ -582,6 +582,7 @@ class slice:
     def indices(self, len: int) -> Tuple[int, int, int]: ...
 
 class tuple(Sequence[_T_co], Generic[_T_co]):
+    def __new__(cls: Type[_T], iterable: Iterable[_T_co] = ...) -> _T[_T_co]: ...
```

Comment:
It looks like `_T[_T_co]` is illegal, although conceptually it's what we want here. We could just return `tuple[_T_co, ...]`, but that would not play well with subclasses.

@ilevkivskyi any ideas on how to spell this type?

Reply:
I think mypy ignores the return type of `__new__` anyway. But IMO the type should be just `(cls: Type[T], iterable: Iterable[_T_co] = ...) -> T`